docs/content/en/docs/reference/clusterspec/optional/proxy.md

```diff
@@ -68,3 +68,6 @@ export NO_PROXY=no-proxy-domain.com,another-domain.com,localhost
    - 192.168.0.0/16
    - .example.com
 ```
+*Note:*
+- For a Bottlerocket OS, it is required to add local IP range in noProxy list.
+- For Bare Metal provider, it is required to host hook images locally which should be accessible by admin machines as well as all the nodes without using proxy configuration. Please refer this documentation for getting URL for hookOS image [here](https://anywhere.eks.amazonaws.com/docs/reference/artifacts/#hookos-kernel-and-initial-ramdisk-for-bare-metal).
```

Comment:
```suggestion
{{% alert title="Note" color="primary" %}}
- For Bottlerocket OS, it is required to add the local subnet CIDR range in the `noProxy` list.
- For Bare Metal provider, it is required to host hook images locally which should be accessible by admin machines as well as all the nodes without using proxy configuration. Please refer to the documentation for getting hook images [here](https://anywhere.eks.amazonaws.com/docs/reference/artifacts/#hookos-kernel-and-initial-ramdisk-for-bare-metal).
{{% /alert %}}
```

Also, can you use relref for the links instead of absolute links. Here's a few examples for reference https://github.com/aws/eks-anywhere/blob/main/docs/content/en/docs/community/docs_contributing.md?plain=1#L53